src/core/boxes.c

```diff
@@ -30,6 +30,12 @@
 #include "util.h"
 #include <X11/Xutil.h>  /* Just for the definition of the various gravities */
 
+void
+meta_rectangle_free (MetaRectangle *meta_rectangle)
```

Comment:
Why `meta_rectangle`? Use `rectangle` or `rect`... Last one seems to be used in this file.

Reply:
Actually why do you need this one? Just use `g_free`?

Reply:
it's a good remark, I will change it to rectangle

Reply:
```C
typedef struct _MetaRectangle MetaRectangle;
struct _MetaRectangle
{
  int x;
  int y;
  int width;
  int height;
};
```
you are right, we can use `g_free` instead of create `meta_rectangle_free` in:
```bash
$ git grep "(GDestroyNotify) meta_rectangle_free"
src/core/boxes.c:  g_list_free_full (filled_list, (GDestroyNotify) meta_rectangle_free);
src/core/edge-resistance.c:  g_slist_free_full (obscuring_windows, (GDestroyNotify) meta_rectangle_free);
```
I will remove `meta_rectangle_free`